mobile/src/tasks/obdTask.ts

```
@@ -0,0 +1,54 @@
+            },
+        );
+
+        while (BackgroundService.isRunning()) {
+            await sleep(5000);
+        }
+
+        obdService.stopTrip();
+        await obdService.stopPolling();
+        locationService.stopLocationTracking();
+        resolve(null);
+    });
```

Biome (2.1.2)

[error] Promise executor functions should not be `async`.



(lint/suspicious/noAsyncPromiseExecutor)